src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.String.cs

```diff
@@ -6,6 +6,21 @@ namespace System.Text.Json.Serialization
 {
     public static partial class JsonSerializer
     {
+        /// <summary>
+        /// Read the UTF-16 encoded JSON <see cref="System.String"/> and return the converted value.
```

Comment:
Previous feedback was to not include the encoding for string and keep that simple.

Compare the following three overloads:
https://github.com/dotnet/corefx/blob/6f970b09b51cb6a266a32242f8b4a6dc7f093740/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs#L15
https://github.com/dotnet/corefx/blob/6f970b09b51cb6a266a32242f8b4a6dc7f093740/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs#L31
https://github.com/dotnet/corefx/blob/6f970b09b51cb6a266a32242f8b4a6dc7f093740/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs#L61

Reply:
I'll remove the "UTF-16" terminology